src/library/scala/Console.scala

```diff
@@ -122,7 +122,7 @@ import scala.util.DynamicVariable
  *             a body of code. Threadsafe by virtue of [[scala.util.DynamicVariable]].
  *
  */
-object Console extends DeprecatedConsole with AnsiColor {
+object Console extends AnsiColor {
```

Comment:
The scaladoc specifically calls out to getting your 22 bizarro ansi escape string literals by importing through `Console`, so I guess this has to go through a deprecation cycle?

>  import Console.{GREEN, RED, RESET, YELLOW_B, UNDERLINED}